app/components/FileList.tsx

```diff
@@ -14,9 +14,25 @@ const FileList = (props: FileListType) => {
     const [files, setFiles] = useState<FileCardType[]>([]);
     const [folders, setFolders] = useState<FolderCardType[]>([]);
 
+    const PERSONAL_SPACE_FOLDER_NAME = "Pesönliche Ablage";
```

Comment:
I think, it should be "Persönliche Ablage" (with an R) ;)

Reply:
Indeed! Changed it